### PR TITLE
add welcome page

### LINKED ISSUE #1: add welcome page
- [x] welcome page  redirects to the next page
- [x] set logo 
- [x] set headline

### PR DESCRIPTION
**Pull request by mewiemers**
Fixes #1

### CODE REVIEW

File: components/welcome/Welcome.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styles from "./Welcome.module.css";
+
+// export type LogoProps = {};
+function Logo() {
+  return (
+    <div className={styles.background}>
+      <h1 className={styles.headline}>Meet</h1>
+      <img src="/logo-albert.svg" />
+
```
Comment (fre-ben) [inline components/welcome/Welcome.tsx]: Remove blank line

**Review by fre-ben — APPROVED** [review]
LGTM 🚀

File: components/welcome/Welcome.module.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  display: grid;
+  justify-items: center;
+  height: 100vh;
+  align-items: center;
+  grid-auto-rows: min-content;
+}
+.headline {
+  font-family: "AlmondCream", sans-serif;
+  color: #707070;
+  font-size: 60px;
```
Comment (m-wiemers) [inline components/welcome/Welcome.module.css]: You should set the size with em :-)

```suggestion
  font-size: 60px;
```

File: components/welcome/Welcome.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  title: "Common/Welcome",
+  component: Welcome,
+} as Meta;
+
+const Template: Story = (args) => <Welcome {...args} />;
+
+export const Primary = Template.bind({});
+Primary.args = {
+  primary: true,
+  label: "Welcome",
```
Comment (m-wiemers) [inline components/welcome/Welcome.stories.tsx]: I don't think you need args if you never use them anyway.

```suggestion
```

**Review by m-wiemers — APPROVED** [review]
If you think perfect is perfect, you've never seen code like this 💻

**Review by RonjaMeinecke — APPROVED** [review]
GJ 🦘

File: styles/globals.css [hunk 4]
```diff
@@ -1,3 +1,8 @@
+@font-face {
+  font-family: "AlmondCream";
+  src: local("AlmondCream"), url(../public/AlmondCream.woff) format("woff");
```
Comment (lmachens) [inline styles/globals.css]: This works, but you'll have AlmondCream.woff two times. 
One time in your public folder `/AlmondCream.woff` and one in `/static/media/AlmondCream.219881f0.woff`.
I would move your fonts to `styles/fonts`.
An alternative is to change the url to access from public folder `url(/AlmondCream.woff)`.

File: public/logo-albert.svg [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  </g>
+  <g id="Gruppe_14" data-name="Gruppe 14" transform="translate(62.733 51.881)">
```
Comment (lmachens) [inline public/logo-albert.svg]: Some of these properties are export artifacts. You can remove them (id, data-name).
It's possible to automatize this by using SVG cleaner/minifier tools.

File: components/welcome/Welcome.tsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./Welcome.module.css";
+import Link from "next/link";
+
+// export type LogoProps = {};
+function Logo() {
+  return (
+    <div className={styles.background}>
+      <h1 className={styles.headline}>Meet</h1>
```
Comment (lmachens) [inline components/welcome/Welcome.tsx]: `h1` should be used only 1 time.

You could think about this structure:

```html
<h1>
  Meet
  <a href="/"><img src="/logo-albert.svg"></a>
  Albert
</h1>
```

File: components/welcome/Welcome.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./Welcome.module.css";
+import Link from "next/link";
+
+// export type LogoProps = {};
```
Comment (lmachens) [inline components/welcome/Welcome.tsx]:
```suggestion
```

File: components/welcome/Welcome.module.css [hunk 8]
```diff
@@ -0,0 +1,13 @@
+.background {
+  background: #50c4b7;
```
Comment (lmachens) [inline components/welcome/Welcome.module.css:2]: If this background is used on other pages too, you could set it in your body.

**Review by lmachens — COMMENTED** [review]
Good job 🚀
Some commit messages doesn't describe what you were doing.

**Pull request merged**